Restsplain.php

```diff
@@ -38,11 +38,13 @@ function get_default_docs_base() {
  * @return string
  */
 function get_docs_base() {
+	$subdir_install = (string) parse_url( home_url(), PHP_URL_PATH );
+
 	if ( get_query_var( 'restsplain', false ) ) {
-		return get_default_docs_base();
+		return $subdir_install . get_default_docs_base();
 	}
 
-	return str_replace( home_url( '/' ), '', get_permalink() );
+	return $subdir_install . str_replace( home_url( '/' ), '', get_permalink() );
```

Comment:
If we know for sure that `$subdir_install` won't have a trailing slash then we should make sure the right hand part of this return value handles that e.g.

```suggestion
	return $subdir_install . '/' . ltrim( str_replace( home_url( '/' ), '', get_permalink() ), '/' );
```

Reply:
Don't we need the same thing on the previous `return` on line 44 then?

Reply:
Yeah that’s a good point. Although the default value has a leading slash it isn’t guaranteed. Can you do that?

Reply:
done